package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "animex",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "dev": "nodemon Server/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/CA-G12/AnimeX.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/CA-G12/AnimeX/issues"
+  },
+  "homepage": "https://github.com/CA-G12/AnimeX#readme",
+  "devDependencies": {
+    "nodemon": "^2.0.19"
+  },
+  "dependencies": {
+    "dotenv": "^16.0.1",
+    "express": "^4.18.1",
+    "jest": "^28.1.3",
+    "node-fetch": "^3.2.10",
+    "supertest": "^6.2.4"
```

Comment:
some packages like `supertest` is used for development sake, so it would make more sense to install them as a devdependencies.

https://www.geeksforgeeks.org/difference-between-dependencies-devdependencies-and-peerdependencies/